bumble/transport/pyusb.py

```diff
@@ -205,16 +206,23 @@ async def close(self):
             await self.sink.stop()
             usb.util.release_interface(self.device, 0)
 
+    device_finder =  usb.core.find
+    if (spec := importlib.util.find_spec('libusb_package')) is not None:
```

Comment:
wouldn't `try: import libusb_package` be simpler here?

Reply:
just a python code style, it's not recommended to use `import` in between functions, only at the top of the script. The importlib is doing the same job to load modules. I don't have strong preference here. I don't need to add the module to `sys`, I can do only a one-time call on the module.

Reply:
The code style recommendation is mainly to avoid local imports of modules when it's not actually needed. But in this case, it really is about trying to import the module locally on purpose, so using `import` is justified. And this way, we're sure that we get the exact standard behavior of `import`, no risk to deviate from it subtly in a way that may be hard to troubleshoot.